### ISSUE TITLE
There's no possibility to force using measurementId from config

### ISSUE THREAD

**Issue by dan-maximov** [firebase/firebase-js-sdk#3971]
### Describe your environment

  * Operating System version: Windows 10
  * Browser version: 86.0.4240.75
  * Firebase SDK version: 7.21.1
  * Firebase Product: analytics

### Describe the problem

I had project where I set measurementId from google analytics, but when I've updated firebase analytics SDK it was replaced with firebase analytics measurementId, so now I have no possibility to track my app as before.

**Comment by hsubox76** [comment]
You can just use plain [gtag](https://developers.google.com/analytics/devguides/collection/gtagjs) to send events you want associated with that measurement ID.

See https://firebase.google.com/docs/analytics/get-started?platform=web#firebase-gtag for details on using gtag and Firebase Analytics simultaneously on the same page. (You won't remove the config line, because, as it says, you don't remove it if you're using a different measurement ID.) If you call `firebase.analytics()` before any gtag calls, these events should be associated with your Firebase project.

**Comment by dan-maximov** [comment]
That makes sense thanks

**Comment by dan-maximov** [comment]
Seems like it doesn't solve this issue. Even if I have gtag initialized for myself, it still uses measurement from the firebase project.
![Untitled](https://user-images.githubusercontent.com/19577962/97230101-06db1180-17e2-11eb-8cb5-119c1fd9d707.png)

**Comment by hsubox76** [comment]
Can I ask what your use case is for Firebase Analytics? Are you hoping to use analytics signals to target users in other Firebase projects such as Remote Config and Messaging? Or do you just need to gather analytics data?

If it's the first case (for use with other Firebase products), you need to use the Firebase measurement ID. If it's the second case, you can use gtag by itself, and don't need Firebase Analytics. You would just go directly to the Analytics dashboard to view usage data (not the Firebase Console).

**Comment by dan-maximov** [comment]
Yes, I'm using firebase analytics to just collect data and watch it on Google Analytics. I don't like to use gtag cause of two reasons:

1. I like to use one tool to deal with all things (firebase auth, firestore, firebase storage, firebase analytics). I was using it all time before this update with forcing measurementId replace.
2. I like the simplicity of firebase analytics, it looks much simpler and stable to just use firebase analitycs bindings for gtag rather than doing it for myself, especially as I already have many firebase tools installed.

**Comment by hsubox76** [comment]
I'm glad you find the Firebase SDK API convenient to use, but one caveat you should know about is that Firebase Analytics is basically a wrapper around gtag that causes all gtag events to await 2 async events, one of which is a network call to get the Firebase measurement ID. This seems like it adds unnecessary complexity and a brief delay (for the network call) in your use case, since you don't care about the Firebase measurement ID or linking it to other Firebase products.

While the gtag API may be less appealing for you (as far as developer experience), it would probably be more efficient functionality-wise, since you avoid making and waiting for unnecessary network requests. (Again, only applicable if you don't want to use Firebase Analytics to target users with Firebase Remote Config or Messaging or other Firebase-related products.)

**Comment by amardeepsingh20** [comment]
I have a related issue. I want to use Firebase Analytics (because I want to use other linked Firebase products) but at the same time I want to utilise cross-domain tracking feature of GA4. 

However, when I Enable Google Analytics in Firebase, it creates a new web stream in Google Analytics which shows "Stream configurations in Firebase" and all the additional options like 'Enhances Measurement' as well as cross-domain linking are not available... 
Is there any way to track users across domains while still using Firebase Analytics?

**Comment by DellaBitta** [comment]
This is an old issue and the modular SDK has shipped at this point. I'm going to close this issue for now but if you still encounter a problem please create a new issue and reference this one. Thanks!
